pkgs/tools/networking/tcptrace/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, libpcap }:
+stdenv.mkDerivation rec {
+  name = "tcptrace";
+  version = "6.6.7";
+  src = fetchurl {
+    url = "http://www.tcptrace.org/download/${name}-${version}.tar.gz";
+    sha256 = "1g8hd6sqwf1f41am5m30kyy0i4wmdzy9ssj7g64s0g4ka500lf33";
+  };
+  patches = [
+    ./debian545595-tcpdump-private-functions.patch
+    ./debian624296-url-decoding.patch
+    ./debian829714-reproducible-builds.patch
```

Comment:
please use fetchpatch for patches instead of vendoring them.

Reply:
Where should I fetchpatch them from?  AFAICT they only live in a source tarball on a Debian mirror.  One of them is mine, written just for this derivation.

Quoth fetchpatch:
```sh
# This is primarily useful for dynamically generated patches,
# such as GitHub's or cgit's, where the non-significant content parts
# often change with updating of git or cgit.
```

Reply:
Specifically they come from: http://deb.debian.org/debian/pool/main/t/tcptrace/tcptrace_6.6.7-5.debian.tar.xz
I guess I could `fetchurl` that as a source and unpack it?

Reply:
We also do this for other package. You can use the systemd package as template.

Reply:
I made the patch tarball a second source instead.